README.md

```diff
@@ -231,8 +231,8 @@ You can use additional tags. Some will be interpreted by *tonic*, others will be
 ### Providing Examples for Structs
```

Comment:
Let's move it under the https://github.com/wI2L/fizz#openapi-specification section, wwhile keeping an anchor link from the "Additional tags" table.